### DISCUSSION TITLE
播放这里的视频 提示灾难性错误

### DISCUSSION THREAD

**Discussion by lv0861** [ywmoyue/biliuwp-lite#652] [bug]
### Bug 描述

https://b23.tv/BV14v4y1U7tr，播放这里的63  64p视频提示灾难性错误 播放不了 别的都可以播放

### 操作步骤

```plain text
https://b23.tv/BV14v4y1U7tr，播放这里的63  64p视频提示灾难性错误 播放不了 别的都可以播放
```

### 期望结果

找到问题解决问题

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

window 10 21h2

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
我这边没复现出来，麻烦提供下日志

**Comment by lv0861** [comment]
刚播放弄得日志，不知道是我安装的问题 还是电脑问题，我也重装下软件试试

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: ywmoyue ***@***.***>
发送时间: Wednesday, May 29, 2024 9:43:58 AM
收件人: ywmoyue/biliuwp-lite ***@***.***>
抄送: lv0861 ***@***.***>; Author ***@***.***>
主题: Re: [ywmoyue/biliuwp-lite] 播放这里的视频 提示灾难性错误 (Discussion #652)


我这边没复现出来，麻烦提供下日志

―

**Comment by lv0861** [comment]
卸载重装后正常，之前覆盖重新安装了也不行，在windows设置里面重置软件了也不行

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: ywmoyue ***@***.***>
发送时间: Wednesday, May 29, 2024 9:43:58 AM
收件人: ywmoyue/biliuwp-lite ***@***.***>
抄送: lv0861 ***@***.***>; Author ***@***.***>
主题: Re: [ywmoyue/biliuwp-lite] 播放这里的视频 提示灾难性错误 (Discussion #652)


我这边没复现出来，麻烦提供下日志

―

**Comment by ywmoyue** [comment]
没收到日志，我猜从邮箱回复github消息时不支持附件

**Comment by lv0861** [comment]
复现的方法是 播放设置里面 有线视频编码选 265的，就会出现这个问题，安装的有HEVC扩展,不过是通过网址下载的那个7元的扩展，以前那个免费的扩展提示不能用，不知道是不是我这扩展的问题

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: ywmoyue ***@***.***>
发送时间: Wednesday, May 29, 2024 11:11:32 AM
收件人: ywmoyue/biliuwp-lite ***@***.***>
抄送: lv0861 ***@***.***>; Author ***@***.***>
主题: Re: [ywmoyue/biliuwp-lite] 播放这里的视频 提示灾难性错误 (Discussion #652)


没收到日志，我猜从邮箱回复github消息时不支持附件

―
